readme.md

```diff
@@ -11,6 +11,8 @@
 [![Build Status][travis-badge]][travis]
 [![Coverage Status][codecov-badge]][codecov]
 [![Hacktoberfest][hacktoberfest-badge]][hacktoberfest]
+[![First timers friendly][first-timers-badge]][first-timers]
+
```

Comment:
Could you remove this extra line (now there’s two blank lines, and the linter doesn’t like that. beep boop 🤖)